### ISSUE TITLE
Hang when shutting down 1.15.2 server

### ISSUE THREAD

**Issue by SonicSpeedster16** [AppliedEnergistics/Applied-Energistics-2#4820]
**Describe the bug**
Issuing the "stop" command on my server causes it to hang after saving the world and results in the process having to be killed.

**To Reproduce**
Run Dedicated Server.
Issue stop command.
Watch sevrer hang

**Expected behavior**
The server to shutdown completely.

**Additional context**
https://gist.githubusercontent.com/SonicSpeedster16/1755c795945e0e09b525a903ce04e54e/raw/6c3bab2df7fd42bc85393f381d03ea5ce4adb19d/gistfile1.txt

**Environment**
- Environment: 
Minecraft 1.15.2
Forge 31.2.45
AE2 7.0.5
Modpack: https://www.curseforge.com/minecraft/modpacks/the-new-adventure/files/3105776

**Comment by ProsperCraft** [comment]
This has been going on since 1.14, I don't think it is related to AE as we had it on my Adventures of Halk pack that didn't have AE2, it continues in to 1.16.3 as well, on Forge and Fabric.

We have to use a custom mod that watches the server and kills it when it fails to completely exit.

**Comment by ProsperCraft** [comment]
Here is our server log from Fabric on a dedicated server with our killserver mod killing the server and showing a stacktrace-
https://gist.github.com/ProsperCraft/7242da265769f53f9f6c3818d42295e3

**Comment by mindforger** [comment]
there is a much more comprehensive logfile that is way smaller :)

> [19:19:13] [Server thread/ERROR]: This crash report has been saved to: /opt/adventuresofhalktwo/./crash-reports/crash-2020-11-09_19.19.13-server.txt

**Comment by yueh** [comment]
We no longer support any minecraft version older than 1.16 at this point.

Should you be able to confirm that this issue still applies to the most recent version, please update the issue and we will reopen it.

**Comment by SonicSpeedster16** [comment]
@yueh I've spent the past couple weeks testing this and truly think at this point that it is a AE2 problem. I've tried downgrading AE2 (and multiple other mods in my pack), but the problem still persists. However, if I remove the Energy Cell and allow my ME network to power down before stopping the server, it shuts down perfectly fine. Replacing the Energy Cell and trying to stop the server while it is powered on results in the server hang. In other words my server shuts down normally unless there is an active ME network.
